june/16/karankartikeya.md

```diff
@@ -0,0 +1,12 @@
+# Yesterday
+- I learned about various online editors like Glitch and Codepen
+- I commented on issue #27 in the nostalgify for my preview of the Navbar
+- I learned about the sticky navbar and fixed navbar
+
+# Today
+- I issued a PR to the User interface for adding the navbar page
```

Comment:
use something like, sent a PR to user Interface to add navbar page.

Reply:
I will make sure of it from next time.